README.md

```diff
@@ -74,11 +81,30 @@ For actually tracing, Tracee needs to run with sufficient capabilities:
 
 Alternatively, running as `root` or with the `--privileged` flag of Docker, is an easy way to start.
 
+#### Linux Headers
+
+In order to compile the eBPF program, Tracee needs some of the Linux kernel headers. Depending on your Linux distribution, there are different ways to obtain it, and different places they end up at.  
+
+> Note that it's important that the kernel headers match the exact version of kernel you are running. To check the current kernel version, run the command `uname -r`
+
+Here are a few examples of how to obtain Linux kernel headers in different Linux distributions:
+
+Distribution | Command | Installation location
+--- | --- | ---
+Ubuntu | apt-get install linux-headers | /usr/bin
+
```

Comment:
thanks @grantseltzer is the kernel-devel actually required? if you have a box to verify this (as well as the install location), that would be great since we want to be sure before adding to the readme

Reply:
pinging @simar7 if you have anything to add here

Reply:
For manjaro it's just /lib/modules/$(uname -r)/build/include/linux without any links

Reply:
how do you install headers on manjaro?

Reply:
sudo pacman -S linux-headers

Reply:
> pinging @simar7 if you have anything to add here

Should we add the section on selinux somewhere? Doesn't have to be here but I feel lots of people will trip considering how it's setup (enforcing) out of the box on certain distros (centos)